cliquet/resource/__init__.py

```diff
@@ -580,8 +580,17 @@ def postprocess(self, result, action=ACTIONS.READ):
         # when it's added
         if action != ACTIONS.READ:
             # the collection has been changed
+            if not isinstance(result, list):
+                result = [result]
```

Comment:
Why do we want it to be a list? It doesn't cope with @tarekziade comment (https://github.com/mozilla-services/cliquet/pull/570/files#r44639830)